db/seeds.rb

```diff
@@ -16,4 +17,8 @@
 Species.find_or_create_by!(name: 'Pelargonia', latin_name: 'Pelargonium', irrigation_level: :low)
 Species.find_or_create_by!(name: 'Agawa', latin_name: 'Agave', irrigation_level: :low)
 Species.find_or_create_by!(name: 'Bonsai', irrigation_level: :high)
-Species.find_or_create_by!(name: 'Bluszcz', latin_name: 'Hedera Helix', irrigation_level: high) 
+Species.find_or_create_by!(name: 'Bluszcz', latin_name: 'Hedera Helix', irrigation_level: :high) 
+
+# Lista naszych roślinek dodanych do aplikacji
+Plant.find_or_create_by!(name: 'Paprotka na oknie', irrigation_frequency_in_days: 2, health_status: :normal, species: Species.third)
```

Comment:
I suggest assigning one of the species above to a variable and them putting it here instead of `Species.third` 😜